dashboard/scripts/update_tts_i18n_static_messages.rb

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env ruby
+require_relative('../config/environment')
+
+# Simple script for creating TTS audio files for some of our clientside
+# feedback strings and UI elements; all those strings for which we
+# generate the TTS URL clientside rather than somewhere in dashboard.
+
+# TODO: (elijah) consider integrating this into the sync-codeout-out
+# script
+
+# TODO: (elijah) formalize a process for flagging these strings somewhere
+# in apps code, rather than maintaining this ugly manual Hash
+
+FEEDBACK_MESSAGES = {
+  "jigsaw": {
+    "reinfFeedbackMsg": "You can press the \"Try again\" button to go back to playing your game."
+  },
+  "studio": {
+    "hoc2015_shareGame": "Share your game:",
+    "shareGame": "Share your story:"
+  },
+  "common": {
+    "emptyBlocksErrorMsg": "The \"Repeat\" or \"If\" block needs to have other blocks inside it to work. Make sure the inner block fits properly inside the containing block.",
+    "emptyFunctionBlocksErrorMsg": "The function block needs to have other blocks inside it to work.",
+    "errorEmptyFunctionBlockModal": "There need to be blocks inside your function definition. Click \"edit\" and drag blocks inside the green block.",
+    "errorGenericLintError": "Your program contains an editor warning that needs to be corrected. Hover over the icons near the line numbers in the editor to learn more.",
+    "errorIncompleteBlockInFunction": "Click \"edit\" to make sure you don't have any blocks missing inside your function definition.",
+    "errorParamInputUnattached": "Remember to attach a block to each parameter input on the function block in your workspace.",
+    "errorQuestionMarksInNumberField": "Try replacing \"???\" with a value.",
```

Comment:
it sure would be nice to, but we don't have translations for "the question marks" just yet. It's on the list.